components/ThemeCard.vue

```diff
@@ -15,7 +15,7 @@ const command = computed(() => {
   <li class="relative w-full h-full flex justify-center items-center p-[1px]">
     <div class="w-full h-full text-xs card rounded-xl z-10 grid grid-cols-1">
       <div class="relative h-[300px]">
-        <img :src="theme.image" class="w-full rounded-t-xl object-cover h-[300px]" alt="" />
+        <NuxtImg :src="theme.image" class="w-full rounded-t-xl object-cover h-[300px]" alt="" sizes="sm:300px lg:600px" width="1112" height="617" format="webp" loading="lazy" />
```

Comment:
ideally we should _conditionally_ set this on indexes 2+ - so `ThemeCard` should take a prop on whether image should be lazy or not.